src/primitives/certificate.cpp

```diff
@@ -198,24 +207,6 @@ bool CScCertificate::IsApplicableToState(CValidationState& state, int nHeight) c
     CCoinsViewCache view(pcoinsTip);
     return view.IsCertApplicableToState(*this, nHeight, state);
 }
-    
-bool CScCertificate::IsStandard(std::string& reason, int nHeight) const
-{
-    if (!zen::ForkManager::getInstance().areSidechainsSupported(nHeight))
-    {
-        reason = "version";
-        return false;
-    }
-
-    // so far just one version is supported, but in future it might not be the case
```

Comment:
Why this check has been removed?

Reply:
version check has been moved to CScCertificate::CheckVersionIsStandard.
CheckOutputsAreStandard has indeed been left out due to faulty refactoring